sorc/link_workflow.sh

```diff
@@ -162,15 +162,19 @@ cd "${HOMEgfs}/parm/ufs" || exit 1
 ${LINK_OR_COPY} "${HOMEgfs}/sorc/ufs_model.fd/tests/parm/noahmptable.tbl" .
 
 cd "${HOMEgfs}/parm/post" || exit 1
-for file in postxconfig-NT-GEFS-F00.txt postxconfig-NT-GEFS.txt postxconfig-NT-GEFS-WAFS.txt \
-    postxconfig-NT-GEFS-F00-aerosol.txt postxconfig-NT-GEFS-aerosol.txt \
-    postxconfig-NT-GFS-ANL.txt postxconfig-NT-GFS-F00.txt postxconfig-NT-GFS-FLUX-F00.txt \
-    postxconfig-NT-GFS.txt postxconfig-NT-GFS-FLUX.txt postxconfig-NT-GFS-GOES.txt \
-    postxconfig-NT-GFS-F00-TWO.txt postxconfig-NT-GFS-TWO.txt \
-    params_grib2_tbl_new post_tag_gfs128 post_tag_gfs65 nam_micro_lookup.dat
+for file in params_grib2_tbl_new nam_micro_lookup.dat
 do
   ${LINK_OR_COPY} "${HOMEgfs}/sorc/upp.fd/parm/${file}" .
 done
+for dir in gfs gefs
+do
+  ${LINK_OR_COPY} "${HOMEgfs}/sorc/upp.fd/parm/${dir}" .
+done
+# Link the post_tag files into parm/post from upp.fd/parm/gfs
+for file in post_tag_gfs128 post_tag_gfs65
+do
+  ${LINK_OR_COPY} "${HOMEgfs}/sorc/upp.fd/parm/gfs/${file}" .
+done
```

Comment:
We don't use these files in the workflow. They are hardwired for a test in the UPP (I think)

Reply:
We do use a few files from `parm/gfs`: `postxconfig-NT-gfs-anl.txt`, `postxconfig-NT-gfs-f00-two.txt`, and `postxconfig-NT-gfs-goes.txt`.

Reply:
Yes.  Those `txt` files are used as the `flatfile`. The `post_tag_*` files are the namelists that are hardwired with little info especially in the `model_inputs` namelist.
We will generate the namelist on the fly until UPP can allow improved flexibility and control over providing input/output files, etc.

Reply:
Oh, misread the lines...  Disregard.